### PR TITLE
Updating and adding more install detail into QuickStart

### PR DESCRIPTION
**Pull request by jbusche**
Have a little more detail on install, cleanup and updated some of the notebook examples steps to match the current notebook demo.
## Merge criteria:

- [ x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by jbusche** [comment]
This PR is for issue 
[image](https://github.com/opendatahub-io/distributed-workloads/issues/62)

File: Quick-Start.md [hunk 1]
```diff
@@ -177,22 +187,60 @@ You can check the status of the Ray cluster and see when its ready to use with:
...
+You can also check the cluster details with:
 
-```python
-! torchx run -s ray -cfg dashboard_address=mnist-head-svc.<Your Namespace>.svc:8265,requirements=requirements.txt dist.ddp -j 2x1 --gpu 1 --script mnist.py
+```Python
+cluster.details()
 ```
 
-Once the job is submitted you can follow it on the Ray dashboard using `cluster.cluster_dash board_uri()` to get the link or `cluster.list_jobs()` and `cluster.job_status(job_id)` to output the job status directly into you're notebook.
+Now the cluster is up.  This guided demo just brings it down again, but in the other guided demos you can follow steps that submit training jobs.
```
Comment (tedhtchang) [inline Quick-Start.md:201]:
```suggestion
Now the cluster is up. This guided demo just brings it down again, but in the other guided demos you can follow steps that submit training jobs.
```

File: Quick-Start.md [hunk 2]
```diff
@@ -177,22 +187,60 @@ You can check the status of the Ray cluster and see when its ready to use with:
...
+
+4. Remove the CodeFlare Operator csv and subscription: (Removes the CodeFlare Operator from the OpenShift Cluster)
+   ```bash
+   kubectl delete sub codeflare-operator -n openshift-operators
+   kubectl delete csv `kubectl get csv -n opendatahub |grep codeflare-operator |awk '{print $1}'` -n openshift-operators
+   ```
+
+5. Remove the CodeFlare CRDs
+   ```bash
+   kubectl delete crd instascales.codeflare.codeflare.dev mcads.codeflare.codeflare.dev schedulingspecs.mcad.ibm.com appwrappers.mcad.ibm.com quotasubtrees.ibm.com
```
Comment (tedhtchang) [inline Quick-Start.md:241]:
```suggestion
   kubectl delete crd instascales.codeflare.codeflare.dev mcads.codeflare.codeflare.dev schedulingspecs.mcad.ibm.com appwrappers.mcad.ibm.com quotasubtrees.ibm.com
```

**Review by tedhtchang — COMMENTED** [review]
Look good just couple of nits

File: Quick-Start.md [hunk 3]
```diff
@@ -44,12 +47,15 @@ NOTE: The above resources are just for the infrastructure pods. To be able to ru
 
 ### OpenShift and Open Data Hub
 
-This Quick Start guide assumes that you have administrator access to an OpenShift cluster and an existing Open Data Hub installation on your cluster. If you do not currently have the Open Data Hub operator installed on your cluster, you can find instructions for installing it [here](https://opendatahub.io/docs/quick-installation/). The default settings for the Open Data Hub Operator will suffice.
+This Quick Start guide assumes that you have administrator access to an OpenShift cluster and an existing Open Data Hub (ODH) installation on your cluster. More information about ODH can be found  [here](https://opendatahub.io/docs/quick-installation/). But the quick step to install ODH is as follows:
+
+1. Using the OpenShift UI, navigate to Operators --> OperatorHub and search for `Open Data Hub Operator` and install it with the default settings
```
Comment (KPostOffice) [inline Quick-Start.md:52]: should be installed with stable or `1.Y.Z`

**Review by KPostOffice — COMMENTED** [review]
Looks good. Please rebase

File: Quick-Start.md (hunk 3 above)
Comment (jbusche) [reply Quick-Start.md]: Thanks Kevin - mentioned `https://www.redhat.com/en/about/company/leadership/chris-wright`

**Review by KPostOffice — APPROVED** [review]
/lgtm

**Pull request merged**